client/src/modules/home/home.css

```diff
@@ -0,0 +1,14 @@
+.home-container {
+  height: 100%;

```

Comment:
overflow auto might be helpful here?

Reply:
I'm thinking about creating a module-container or class that defaults to height: 100%, overflow: auto -> thoughts?

Reply:
I looked up overflow: auto, but not sure the impact it will have on appearance in this case. you?
The container class def makes sense.